parachains/runtimes/assets/statemine/src/weights/pallet_xcm.rs

```diff
@@ -62,10 +62,8 @@ impl<T: frame_system::Config> pallet_xcm::WeightInfo for WeightInfo<T> {
 		//  Measured:  `75`
 		//  Estimated: `9780`
 		// Minimum execution time: 28_004_000 picoseconds.
-		Weight::from_parts(28_341_000, 0)
-			.saturating_add(Weight::from_parts(0, 9780))
-			.saturating_add(T::DbWeight::get().reads(5))
-			.saturating_add(T::DbWeight::get().writes(2))
+		Weight::from_parts(0, 0)
```

Comment:
I will remove these weight hacks in an upcoming PR to use the `bko-transfer-asset-via-bridge`.